itl/plugins-contrib.d/network-components.conf

```diff
@@ -604,7 +604,7 @@ object CheckCommand "nwc_health" {
 			description = "The role of this device in a hsrp group (active/standby/listen)"
 		}
 		"--report" = {
-			set_if = "$nwc_health_report$"
+			value = "$nwc_health_report$"
```

Comment:
values can be "long" (default), "short" (to shorten if available), or "html" (to produce some html outputs if available)